iosApp/fastlane/Snapfile

```diff
@@ -0,0 +1,35 @@
+# Uncomment the lines below you want to change by removing the # in the beginning
+
+# A list of devices you want to take the screenshots from
+devices([
+  "iPhone 8",
```

Comment:
Maybe we should check SE, since it’s the smallest screen supported?

Reply:
I'll change that in #34